crates/typst/src/model/list.rs

```diff
@@ -85,7 +85,13 @@ pub struct ListElem {
     /// - Items
     /// ```
     #[borrowed]
-    #[default(ListMarker::Content(vec![TextElem::packed('•')]))]
+    #[default(ListMarker::Content(
+        vec![
+        TextElem::packed('\u{2022}'),
+        TextElem::packed('\u{25E6}'),
+        TextElem::packed('\u{2023}'),
+        ]
```

Comment:
Given the points made in https://github.com/typst/typst/issues/1520#issuecomment-1826826247, I would propose the order `•` (U+2022), `‣` (U+2023), `–` (U+2013). This order circumvents U+25AA which is not centered in the default font. It also arranges the symbols in decreasing order of visual weight. Finally, it does not contain U+25E6 which both @laurmaedje and I strongly dislike.

I do not feel like we have to copy the U+2022, U+25E6, U+25AA order from Word et al. LibreOffice Writer and Google Docs converged on this because they try to be just like Word while we hardly even try to be just like LaTeX. We are free to choose our enum symbols in a way which we believe to be conducive to effective and aesthetic communication.

That being said, if we must use U+25E6 due to a strong consensus by other contributors / users, it would replace the en-dash in my order since it is only an outline instead of a solid shape.